tests/Geppetto/test/campaigns/files/generated_files/download/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
```

Comment:
add an empty `.gitkeep` file instead and configure the .gitignore file in `tests/gepetto`